src/monitoring_service/handlers.py

```diff
@@ -53,10 +53,15 @@ def latest_committed_block(self) -> BlockNumber:
         has been written to the DB. """
         return self.ms_state.blockchain_state.latest_committed_block
 
+    @property
+    def latest_block(self) -> BlockNumber:
```

Comment:
Should we call this `latest_unconfirmed_block` to make sure that it is not misunderstood?

If `self.web3.eth.blockNumber` does an RPC call, I am against making it a property, since we expect attributes to be fast, non-blocking, non-switching.